audbcards/core/datacard.py

```diff
@@ -339,6 +339,82 @@ def save(self, file: str = None):
             with open(file, mode="w", encoding="utf-8") as fp:
                 fp.write(self.content)
 
+    @property
+    def segment_duration_distribution(self) -> str:
+        r"""Minimum and maximum of segment durations, and plotted distribution.
+
+        This generates a single line
+        containing the mininimum and maximum values
+        of segment durations.
+
+        If :attr:`audbcards.Datacard.sphinx_src_dir` is not ``None``
```

Comment:
Thanks for elaborating about the  documentation file paths and the meaning of sphinx-dir. so you are describing the self-managed `sphinx-src-dir`  and nothing that the user is controlling. 

I wonder whether the docstring here should forward this information to  the reader of the docstring here?

Reply:
The user has control of `audbcards.Datacard.sphinx_src_dir` as this is an argument of `audbcards.Datacard`. From `audbcards.Datacard`'s docstring:

![image](https://github.com/user-attachments/assets/d618dd2c-283f-4984-85a5-efafbbd7e0b8)

![image](https://github.com/user-attachments/assets/8e643399-dbc0-431e-bd21-720e43d6f8ab)

Should we highlight, that the user can set `sphinx_src_dir` when initializing `audbcards.Datacard`?

Reply:
Possibly. The aspsects that got me confused were that (i)  after building the documentatioan `sphinx_src_dir` 
 directory will be going back to empty as the audbcards sphinx app cleans it up, (ii) and that the results in `build/html` obey to how sphinx structures the documentation, and do not relate to the docstring that talks about `sphinx_src_dir`.

Reply:
Yes, one reason for confusion is that the sphinx extension takes care of cleaning up after the HTML pages are built.
But this is not the case, if you just play around with the `audbcards.Datacard` yourself, e.g.
```python
import tempfile

import audbcards
import audeer


with tempfile.TemporaryDirectory() as sphinx_src_dir:
    with tempfile.TemporaryDirectory() as cache_root:
        ds = audbcards.Dataset("vadtoolkit", "1.1.0", cache_root=cache_root)
        dc = audbcards.Datacard(ds, sphinx_src_dir=sphinx_src_dir, cache_root=cache_root)
        dc.segment_duration_distribution
        print("Content of 'sphinx_src_dir'")
        print(audeer.list_file_names(sphinx_src_dir, recursive=True))
        print("Content of 'cache_root'")
        print(audeer.list_file_names(cache_root, recursive=True))
```
which returns
```
Content of 'sphinx_src_dir'
['/tmp/tmp832uln4n/datasets/vadtoolkit/vadtoolkit-1.1.0-segment-duration-distribution.png']
Content of 'cache_root'
['/tmp/tmp80umam8s/vadtoolkit/1.1.0/vadtoolkit-1.1.0-segment-duration-distribution.png', '/tmp/tmp80umam8s/vadtoolkit/1.1.0/vadtoolkit-1.1.0.pkl']
```

BTW, if a database has no segments, those files will also not be created
```python
import tempfile

import audbcards
import audeer


with tempfile.TemporaryDirectory() as sphinx_src_dir:
    with tempfile.TemporaryDirectory() as cache_root:
        ds = audbcards.Dataset("emodb", "1.4.1", cache_root=cache_root)
        dc = audbcards.Datacard(ds, sphinx_src_dir=sphinx_src_dir, cache_root=cache_root)
        dc.segment_duration_distribution
        print("Content of 'sphinx_src_dir'")
        print(audeer.list_file_names(sphinx_src_dir, recursive=True))
        print("Content of 'cache_root'")
        print(audeer.list_file_names(cache_root, recursive=True))
```
returns
```
Content of 'sphinx_src_dir'
[]
Content of 'cache_root'
['/tmp/tmpjh_lxbrp/emodb/1.4.1/emodb-1.4.1.pkl']
```

Reply:
I extended the docstring to mention, that the dataset needs to contain segments:

![image](https://github.com/user-attachments/assets/db62d1b5-0a16-431d-b2ed-4924fdf97356)

I also updated the docstring of `audbcards.Datacrad.file_duration_distribution` accordingly:

![image](https://github.com/user-attachments/assets/392d1a9a-74ad-4b9f-8128-8f0c709d3a8f)

Reply:
>  The aspsects that got me confused were that (i) after building the documentatioan `sphinx_src_dir`
directory will be going back to empty as the `audbcards` sphinx app cleans it up, (ii) and that the results in `build/html` obey to how sphinx structures the documentation, and do not relate to the docstring that talks about `sphinx_src_dir`.

I think this is not that important, because when you use the sphinx extension, you will not call `audbcards.Datacard` yourself. But maybe we should extend https://audeering.github.io/audbcards/sphinx-extension.html explaining which files are generated and deleted.

Reply:
I created https://github.com/audeering/audbcards/issues/105 and will mark this as resolved here.